src/main/java/bobby/Parser.java

```diff
@@ -0,0 +1,43 @@
+package bobby;
+
+import bobby.command.ByeCommand;
+import bobby.command.Command;
+import bobby.command.DeadlineCommand;
+import bobby.command.DeleteCommand;
+import bobby.command.EventCommand;
+import bobby.command.FindCommand;
+import bobby.command.InvalidCommand;
+import bobby.command.ListCommand;
+import bobby.command.MarkCommand;
+import bobby.command.ToDoCommand;
+import bobby.command.UnmarkCommand;
+
+public class Parser {
+
+    public static Command parse(String fullCommand) {
+        String[] fullCommandArr = fullCommand.split(" ");
+        Command command;
+        switch(fullCommandArr[0].toLowerCase()) {
```

Comment:
Really good in remembering to follow the module's switch statement standard...same goes for other switch statements. 👍